app/actions/todos.go

```diff
@@ -20,7 +20,20 @@ func Index(c buffalo.Context) error {
 	return c.Render(http.StatusOK, r.HTML("todos/index.plush.html"))
 }
 
-func NewTodo(c buffalo.Context) error  {
+func NewTodo(c buffalo.Context) error {
 	c.Set("todo", models.Todo{})
 	return c.Render(http.StatusOK, r.HTML("todos/new.plush.html"))
 }
+
+func StoreTodo(c buffalo.Context) error {
+	tx := c.Value("tx").(*pop.Connection)
+	todo := models.Todo{}
+	todo.IsCompleted = false
```

Comment:
When declaring a new Todo, this field is set to false by default.
We can remove this line.

Reply:
I think that I didn't set the field `is_completed` as false by default, do I need create another migration ?

Reply:
Done!